src/main/java/jupyter/Displayers.java

```diff
@@ -57,8 +57,10 @@ public static void setMimeTypes(String... types) {
    */
   @SuppressWarnings("unchecked")
   public static <T> Map<String, String> display(T obj) {
-    return registration()
-        .find((Class<T>) obj.getClass())
-        .display(obj);
+    Displayer<? super T> displayer = registration().find((Class<T>) obj.getClass());
+    if (displayer != null)
```

Comment:
Nit: Please use `}` and `}` regardless of the length of the following block.

Reply:
Should be fixed.